Tasks/NpmAuthenticateV0/npmauth.ts

```diff
@@ -121,7 +121,7 @@ async function main(): Promise<void> {
 
 
 main().catch(error => {
-    tl.rmRF(util.getTempPath());
+    tl.rmRF(tl.getVariable("NPM_AUTHENTICATE_TEMP_DIRECTORY"));
```

Comment:
I think this is not guranteed to be set at this point. What if the process fails on line 35?

Reply:
Fair point, I'll check that it's set to something first.